Dockerfile

```diff
@@ -0,0 +1,69 @@
+FROM node:18-alpine
+WORKDIR /app
+
+# Do all of these first so Docker can cache the result for faster DEV cycles
+# (we can "build" (from cache) in 2 seconds instead of 5 minutes)
+COPY package.json package.json
```

Comment:
package-lock.json should also be copied as it contains the metadata needed to make sure the correct versions are installed of all packages

Reply:
done, thanks